package.json

```diff
@@ -68,7 +68,7 @@
   },
   "author": "engineering@strongloop.com",
   "license": {
-    "name": "Dual Artistic/StrongLoop",
+    "name": "Artistic-2.0 OR DocumentRef-LICENSE.md:LicenseRef-StrongLoop",
```

Comment:
It doesn't match the syntax described in the npm package docs, and npm install warns about it. Your probably need a combination of

       If  your  package  is  licensed  under  multiple common licenses, use an SPDX license
       expression syntax version 2.0 string http://npmjs.com/package/spdx, like this:

         { "license" : "(ISC OR GPL-3.0)" }

and

       If you are using a license that hasn't been assigned an SPDX identifier,  or  if  you
       are using a custom license, use the following valid SPDX expression:

         { "license" : "LicenseRef-LICENSE" }

       Then include a LICENSE file at the top level of the package.